tutorials/general/t8_step4_partition_balance_ghost.cxx

```diff
@@ -281,7 +281,7 @@ t8_step4_main (int argc, char **argv)
     (" [step4] Repartitioned forest and built ghost layer.\n");
   t8_step3_print_forest_information (forest);
   /* Write forest to vtu files. */
-  t8_forest_write_vtk (forest, prefix_partition_ghost);
+  t8_forest_write_vtk_ext (forest, prefix_partition_ghost, 1, 1, 1, 1, 1, 0, 0, 0, NULL);
```

Comment:
```suggestion
  t8_forest_write_vtk_ext (forest, prefix_partition_ghost, 1, 1, 1, 1, 1, 0, 1, 0, NULL);
```
Since the export of ghost elements is not supported with the VTK API, we have to disable it